cli/cli
# update(repo create): Support for `--homepage` flag for template-based repository creation flow

Fixes https://github.com/cli/cli/issues/7934

This PR adds support for:

- Using `--homepage` flag to set the repository's homepage URL in the template-based repository creation flow

Similar to the work in [Disable wiki with template](https://github.com/cli/cli/pull/7886) and [Disable issues with template](https://github.com/cli/cli/pull/7918), this is done using [updateRepository](https://docs.github.com/en/graphql/reference/mutations#updaterepository) after cloning repo from template repo.

Testing instructions:

- With this branch checked out locally, run `repo create <repo-name> --template <template-repo-owner>/<template-repo-slug> --private --disable-issues --homepage "https://example.com"`
- Homepage URL should be set properly 

Test template to try with: https://github.com/moul/golang-repo-template
